gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -123,7 +127,7 @@ object ConverterUtils extends Logging {
   // TODO: This is used only by `BasicScanExecTransformer`,
   //  perhaps we can remove this in the future and use `withExprId` version consistently.
   def collectAttributeNamesWithoutExprId(attributes: Seq[Attribute]): JList[String] = {
-    collectAttributeNamesDFS(attributes)(genColumnNameWithoutExprId)
+    collectAttributeNamesDFS(attributes)(getAttributeName)
```

Comment:
does it affect ch backend ? cc @zzcclp

Reply:
In `genColumnNameWithoutExprId`, it will call `getShortAttributeName ` to remove the `(xxx` part, I am not sure whether it will affect ?

Reply:
`// TODO: This is used only by BasicScanExecTransformer,`

CH CI has passed, so this change does not affect ch backend.


> In genColumnNameWithoutExprId, it will call getShortAttributeName to remove the (xxx part,

I found this logic and does not change it.